README.rst

```diff
@@ -50,11 +50,43 @@ Basic mode
    pip install -e .
 
 Dev mode
-^^^^^^^^^^
+^^^^^^^^
 .. code:: sh
 
    pip install -e .[dev]
 
+Version control of database using DVC (Data Version Control)
+-------------------------------------------------------------
+
+Data Version Control (DVC) is a data management tool that is meant to be run alongside Git.
+In this project, DVC is used to link changes in the code to specific versions of a sample database containing example project files.
+DVC can be used when this project is installed in Dev mode. You can read more about DVC and how to use it `here <https://dvc.org/doc/start>`_.
+**Note:** Remote access to the sample database stored on google drive is currently restricted. Access requires a :code:`gdrive_client_secret`
+for user access authentication to be shared by developers.
+
+DVC usage
+~~~~~~~~~
+
+To download the sample database:
+
+.. code:: sh
+
+   git pull
+   dvc remote modify shared_gdrive gdrive_client_secret <client_secret_here> # Contact developers for grive_client_secret
```

Comment:
I was looking for a way to prevent the secret from affecting a file that is tracked by git, because it seemed to easy for a developer to accidentally commit it in the future (the secret would end up stored inside `.dvc/config` which would show up as a change to potentially be committed). I found that if you add `--local` like this the key will instead go into `.dvc/config.local` which is already in `.dvc/.gitignore` so the danger goes away

```suggestion
   dvc remote modify --local shared_gdrive gdrive_client_secret <client_secret_here> # Contact developers for grive_client_secret
```